Dockerfile

```diff
@@ -1,36 +1,3 @@
-FROM google/dart:2.4.0
-WORKDIR /build/
+FROM drydock-prod.workiva.net/workiva/dart_build_image:1 as build
 
-# Build Environment Vars
-ARG BUILD_ID
-ARG BUILD_NUMBER
-ARG BUILD_URL
-ARG GIT_COMMIT
-ARG GIT_BRANCH
-ARG GIT_TAG
-ARG GIT_COMMIT_RANGE
-ARG GIT_HEAD_URL
-ARG GIT_MERGE_HEAD
-ARG GIT_MERGE_BRANCH
-# Expose env vars for git ssh access
-ARG GIT_SSH_KEY
-ARG KNOWN_HOSTS_CONTENT
-
-# Install SSH keys for git ssh access
-RUN mkdir /root/.ssh
-RUN echo "$KNOWN_HOSTS_CONTENT" > "/root/.ssh/known_hosts"
-RUN echo "$GIT_SSH_KEY" > "/root/.ssh/id_rsa"
-RUN chmod 700 /root/.ssh/
-RUN chmod 600 /root/.ssh/id_rsa
-
-RUN pub global activate --hosted-url https://pub.workiva.org semver_audit ^2.0.1
-
-COPY pubspec.yaml ./
-RUN pub get
-
-ADD ./ ./
-
-RUN pub global run semver_audit report --repo Workiva/json_schema
```

Comment:
I assume the new base image does this ^^?

Reply:
That's the only reason we still had an internal build vs travis

Reply:
Yeah the dart_build_image handles pretty much everything now!

Reply:
I'm not sure that's what we want. Since this is a public repo, we want to do everything in the open, on Travis, rather than on Workiva build.